tests/simulator_test.cpp

```diff
@@ -119,9 +110,8 @@ TEST_F(EnergyCollectorTest, NotEvenNumOfEnergyCollectorTest) {
   RayHitData hitData;
   ASSERT_TRUE(performHitCollector(energyCollectors, straightUp, &hitData));
 
-  const float collectorPositionRadius = 4;
-  const float collectorAngle = 2 * kPi / (numCollectors - 1);
   const float refCollectorRadius = energyCollectors[0]->getRadius();
+  const float collectorPositionRadius = 4;
   Vec3 collisionPointStraightUp =
       Vec3(0, 0, collectorPositionRadius - refCollectorRadius);
   ASSERT_EQ(hitData.collisionPoint(), collisionPointStraightUp);
```

Comment:
Here.

Reply:
Not fixed. Did not seem like anything changed in this line?

Reply:
let me know if its what you meant ;)